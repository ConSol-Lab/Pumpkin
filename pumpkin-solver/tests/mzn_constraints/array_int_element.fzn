var 1..3: index :: output_var;
var 1..6: rhs :: output_var;

constraint array_int_element(index, [3, 5, 6], rhs);

solve satisfy;
