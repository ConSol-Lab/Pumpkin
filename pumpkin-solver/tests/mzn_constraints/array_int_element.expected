index = 1;
rhs = 3;
----------
index = 2;
rhs = 5;
----------
index = 3;
rhs = 6;
----------
==========
